actions.md

```diff
@@ -0,0 +1,547 @@
+- An action runs for a bounded amount of time. This limit can be configured per action, and applies to both the initialization and the execution separately. If the action time limit is exceeded during the initialization or run phase, the activation's response status is _action developer error_.
+
+## Other action commands
+
+### Get metadata for actions
+
+Metadata that describes existing actions can be retrieved via the `nim action get` command.
+
+**[[NH: check output, looks like the original had no package so my command is wrong?]]**
+```
+nim action get greeting
+ok: got action greeting
+{
+    "namespace": "guest",
+    "name": "hello",
+    "version": "0.0.1",
+    "exec": {
+        "kind": "nodejs:6",
+        "binary": false
+    },
+    "annotations": [
+        {
+            "key": "exec",
+            "value": "nodejs:6"
+        }
+    ],
+    "limits": {
+        "timeout": 60000,
+        "memory": 256,
+        "logs": 10
+    },
+    "publish": false
+}
+```
+
+### Get the URL for an action
+
+An action can be invoked through the REST interface via an HTTPS request.
+
+To get an action URL, execute the following command:
+
+```
+nim action get greeting --url
+```
+
+A URL with the following format will be returned for standard actions:
+```
+ok: got action actionName
+https://${APIHOST}/api/v1/namespaces/${NAMESPACE}/actions/greeting
+```
+
+**[[NH: check link]]**
+Authentication is required when invoking an action via an HTTPS request using this resource path. For more information regarding action invocations using the REST interface, see [Using REST APIs with OpenWhisk](rest_api.md#actions).
+
+**[[NH: check link:]]**
+Another way of invoking an action that does not require authentication is via [web actions](webactions.md#web-actions).
+
+Any action may be exposed as a web action, using the `--web true` command line option when creating or updating an action.
+
+```
+nim action update greeting --web true
+ok: updated action greeting
+```
+
+The resource URL for a web action is different:
+```
+nim action get greeting --url
+ok: got action greeting
+https://${APIHOST}/api/v1/web/${NAMESPACE}/${PACKAGE}/greeting
+```
+
+You can use `curl` or `wget` to invoke the action.
+```
+curl `nim action get greeting --url | tail -1`.json
+{
+  "payload": "Hello, Toto from somewhere!"
+}
+```
+
+### Save action code
+
+Code associated with an existing action may be retrieved and saved locally. Saving can be performed on all actions except sequences and docker actions.
+
+1. Save action code to a filename that corresponds with an existing action name in the current working directory. A file extension that corresponds to the action kind is used. An extension of _.zip_ is used for action code that is a zip file.
+
+  **[[NH: please check command and output:]]**
```

Comment:
`nim` does not display any command line output here (it just returns, having saved the code as you requested)

On the general question of what to do about the fact that we do not provide a /whisk-system/samples package for customers to experiment with, I will let @rabbah comment.

Reply:
these samples are useful for primiarly one thing, saving the user from creating boiler plate functions for the sake of the examples in the docs.

(this once led to the idea to inline functions on the command line :)
if we had a way to generate samples from the command line (nim project create? this would not necessary i think - in the overall doc flow which introduces projects before individual actions, this may work).